src/Application/Rsp.IrasService.Application/CQRS/Queries/GetDocumentsForModificationQuery.cs

```diff
@@ -0,0 +1,23 @@
+﻿using MediatR;
+using Rsp.IrasService.Application.DTOS.Requests;
+using Rsp.IrasService.Application.DTOS.Responses;
+
+namespace Rsp.IrasService.Application.CQRS.Queries;
+
+public class GetDocumentsForModificationQuery
+(
+    Guid modificationId,
+    ProjectOverviewDocumentSearchRequest searchQuery,
+    int pageNumber,
+    int pageSize,
+    string sortField,
+    string sortDirection
+) : IRequest<ProjectOverviewDocumentResponse>
+{
+    public Guid ModificationId { get; } = modificationId;
+    public ProjectOverviewDocumentSearchRequest SearchQuery { get; } = searchQuery;
+    public int PageNumber { get; } = pageNumber;
+    public int PageSize { get; } = pageSize;
+    public string SortField { get; } = sortField;
+    public string SortDirection { get; } = sortDirection;
+}
```

Comment:
No need to define the parameters in constructor and assign them again to the properties. Just define the properties and mark them as required for the properties that are mandatory. Then in the controller just initialize the command using object initializer syntax.

You may need to update CodeMaid settings so that you can add `required`


```suggestion
public class GetDocumentsForModificationQuery : IRequest<ProjectOverviewDocumentResponse>
{
    public required Guid ModificationId { get; } = modificationId;
    public required ProjectOverviewDocumentSearchRequest SearchQuery { get; } = searchQuery;
    public required int PageNumber { get; } = pageNumber;
    public required int PageSize { get; } = pageSize;
    public required string SortField { get; } = sortField;
    public required string SortDirection { get; } = sortDirection;
}
```